automat/_test/test_visualize.py

```diff
@@ -44,8 +62,142 @@ def out(self):
     return mm
 
 
+class ElementMakerTests(TestCase):
+    """
+    Tests that ensure elementMaker generates correct HTML.
```

Comment:
The @jml –standard formulation of this would be something like "`L{elementMaker} generates HTML representing an element.`"